aries-site/src/pages/foundation/typography.mdx

```diff
@@ -87,6 +132,22 @@ When implementing heading levels, always use the correct semantic heading level.
   <TextSizeExample />
 </Example>
 
+### Text best practices
+
+Use the t-shirt size `maxWidth` design token on paragraphs to limit the width of text to ensure readability.
+
+While each text t-shirt sizes comes with a default font weight, the emphasis of text can be adjusted to suit the context by using the [fontWeight](/design-tokens/global#fontweight) design tokens.
+
+## Responsiveness
+
+The values of heading tokens automatically step down one t-shirt size at the `small` [breakpoint](/design-tokens/global#breakpoint). This ensures content scales appropriately as the screen size decreases.
+
+When utilizing the CSS files, a media query is included in the "dimension.small.css" file to adjust the token values at the `small` breakpoint.
```

Comment:
This is too specific to a particular implementation.

I would move this a content under `design-tokens/global#breakpoint` either as inline content or as a "Tip" inline notification (but @julauxen is trying to avoid the latter).